engine/src/test/java/org/camunda/bpm/engine/test/bpmn/scripttask/ScriptTaskGraalJsTest.java

```diff
@@ -49,9 +49,11 @@
   private static final String GRAALJS = "graal.js";
 
   protected ScriptEngineResolver defaultScriptEngineResolver;
+  protected boolean spinEnabled = false;
 
   @Before
   public void setup() {
+    spinEnabled = processEngineConfiguration.getEnvScriptResolvers().stream().anyMatch(resolver -> resolver.getClass().getSimpleName().equals("SpinScriptEnvResolver"));
```

Comment:
Can we put some of the code on a new line to make it more readable. The max line length was 120 now right?

Reply:
Yes you're right, I tend to overlook that due to a big monitor setup 😁

Reply:
```suggestion
    spinEnabled = processEngineConfiguration.getEnvScriptResolvers().stream()
                                .anyMatch(resolver -> resolver.getClass().getSimpleName().equals("SpinScriptEnvResolver"));
```